### ISSUE TITLE
Split inline bash code into own script(s)

### ISSUE THREAD

**Issue by jcbhmr** [spenserblack/actions-wiki#23]
Discussed in #19

**Comment by spenserblack** [comment]
>In the future, maybe use a JS action or a Docker thing or some other technique.

I'll say now that, if we decide that a composite action isn't enough, I'd want to make a JS action.

## Pros

- Can be used with jobs that run on Windows
- Docker actions are often slow (it builds from the `Dockerfile` *each run*) unless the docker image is published (technically a Docker con, not a JS pro)

**Comment by jcbhmr** [comment]
btw as proof (for me at least; id never seen this) you CANNOT run a Docker container-based image on Windows (at least not right now)

![image](https://user-images.githubusercontent.com/61068799/225465716-f9699a6c-d94f-4e4a-8f98-4077c8e98ed5.png)

**Comment by spenserblack** [comment]
Yeah, AFAIK it's a licensing issue. As it is, GH either can't or doesn't want to use docker on Windows commercially.
